sphinx/registry.py

```diff
@@ -11,7 +11,7 @@
 if sys.version_info >= (3, 10):
     from importlib.metadata import entry_points
 else:
-    from importlib_metadata import entry_points
+    from importlib_metadata import entry_points  # type: ignore[import-not-found]
```

Comment:
this shouldn't be required. `mypy` understands `if sys.version_info ...` calls
if this is throwing an error, maybe `(3, 10)` is the wrong version?

Reply:
No, it's the correct one. `importlib.metadata` was experimental even though it was present in the standard library. In addition, entry_points were different:

```
The “selectable” entry points were introduced in importlib_metadata 3.6 and Python 3.10. Prior to those changes, entry_points accepted no parameters and always returned a dictionary of entry points, keyed by group. With importlib_metadata 5.0 and Python 3.12, entry_points always returns an EntryPoints object. See [backports.entry_points_selectable](https://pypi.org/project/backports.entry-points-selectable) for compatibility options.
```

(see "Compatibility note" of https://docs.python.org/3/library/importlib.metadata.html)